requirements.txt

```diff
@@ -2,10 +2,8 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --output-file=requirements.txt
+#    pip-compile
 #
-colorama==0.4.6
```

Comment:
Curious why colorama is now deleted when none of the dependencies have changed. Is this to do with running pip-compile on windows vs mac? In which case maybe we should have separate requirements files for windows/mac/linux?

Reply:
I don't know, to be honest. All tests are passing in all the platforms so if this were something that needs to be different for each of them, I would expect for some of the tests to fail if a platform requires something that has not been installed.

Reply:
Looks like it's needed to create coloured terminal text in Windows, but everything seems fine without it on my machine